smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -49,6 +49,7 @@ be used.
                             switch form uses local IP address as hostname
   --username                anonymous, or Key Vault if --registry is specified
   --no-deployment           deploy Edge Hub and temperature sensor modules
+  --deployment              deployment json file
 "
```

Comment:
Here it should be the description for the default, something like: deploy Edge Hub and temperature sensor module, or no deployment if --no-deployment is set

Reply:
This is the option for the deployment file.